chainer/variable.py

```diff
@@ -82,6 +82,18 @@ def __len__(self):
         """
         return self.data.size
 
+    @property
+    def label(self):
+        """Returns string that represents its property
+
+        Returns:
+            string: represents its shape and dtype
```

Comment:
I think the return value of a property should be described at the summary, so how about removing this section?